charts/alertmanager/templates/statefulset.yaml

```diff
@@ -28,6 +28,14 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args: 
+            - --storage.path=/alertmanager
+            {{- if .Values.configFileName }}
```

Comment:
Instead of add a configFilename value, if the purpose is to omit the config file, can we insead check `{{- if .Values.conig }}` ? i.e. if we set `config: null` in the values file, it will not pass it in.

Reply:
I'm not sure that I fully understand.
Do you mean that we have to check `{{- if .Values.config }}` and if it empty skip --config.filename ?
I'm not sure that it is possible because it will cause alertmanager to start finding config in local dir (default config path) and it will fail. To avoid it we will have to add some default behavior as some new helper or hardcode default value inplace.

I guess we should check {{- if .Values.config }} and if it is empty do not install ConfigMap, Volume, and skip corresponding VolumeMount. In my opinion, it is more predictable behavior.

Reply:
ah yeah that makes sense. Maybe we skip the evaluation and always use the config file instead, that will be simpler